todoapp-main/build.gradle

```diff
@@ -13,6 +13,7 @@ repositories {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-jdbc'
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.16.1'
```

Comment:
Whats that for? I removed form dependency and nothing changed

Reply:
This dependency improve datetime deserialization with jackson, this library takes an effect only in runtime.